opencga-app/app/scripts/azure/arm/appgw/functions.json

```diff
@@ -0,0 +1,154 @@
+{
+    "$schema": "http://schemas.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "functionAppName": {
+        "type": "string"
+      },
+      "location": {
+        "type": "string",
+        "defaultValue": "[resourceGroup().location]",
+        "metadata": {
+          "description": "Location for all resources."
+        }
+      },
+      "appGwDomain": {
+        "type": "string",
+        "metadata": {
+          "description": "FQDN of application gateway (url to get SSL certificate for)"
+        }
+      },
+      "appGwName": {
+        "type": "string",
+        "defaultValue": "appgw",
+        "metadata": {
+          "description": "Resource name of application gateway"
+        }
+      },
+      "appGwRg": {
+        "type": "string",
+        "defaultValue": "[resourceGroup().name]",
+        "metadata": {
+          "description": "Resource group name where application gateway is deployed."
+        }
+      },
+      "appGwSslEmail": {
+        "type": "string",
+        "defaultValue": "example@domain.com",
```

Comment:
Is this an OK default? Is it better to remove the default (and fail if it's missing)? If you really need a default, should it be something @example.com (is domain.com doesn't look like a domain that can be used for a placeholder)

Reply:
Agreed, accidentally left it there when testing. defaultValue removed in https://github.com/opencb/opencga/pull/1202/commits/40a78ca90e6f746f8bf632fb7dbe302d7f626c70